modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -154,6 +154,9 @@ public class PersistentPageMemory implements PageMemory {
     /** Try again tag. */
     public static final int TRY_AGAIN_TAG = -1;
 
+    /** Threshold of the checkpoint buffer. We should start forcefully checkpointing its pages upon exceeding it. */
+    private static final float CP_BUF_FILL_THRESHOLD = 2.0f / 3;
```

Comment:
Why exactly this value, and not, say, 0.75? 
Can you indicate in the documentation why this value was chosen.

Reply:
I copied it from Ignite 2.x, so I don't know why exactly it was chosen

Reply:
Please indicate that the value is taken from ignite 2.0 in the documentation.

Reply:
Sure